packages/web-components/components/selfie/src/selfie-capture/SelfieCapture.js

```diff
@@ -557,12 +561,16 @@ class SelfieCaptureScreen extends HTMLElement {
     this.render = () => this.templateString();
 
     this.attachShadow({ mode: 'open' });
+    this.facingMode = 'user';
+    if (this.allowAgentMode) {
+      this.facingMode = 'environment';
+    }
   }
 
   connectedCallback() {
     const template = document.createElement('template');
     template.innerHTML = this.render();
-
+    this.shadowRoot.innerHTML = '';
```

Comment:
why did we need to do this?

Reply:
This is so that the view will be reset when the component is rerendered